cmd/site-replication.go

```diff
@@ -641,6 +641,8 @@ func (c *SiteReplicationSys) validateIDPSettings(ctx context.Context, peers []Pe
 
 	for i := 1; i < len(s); i++ {
 		if !reflect.DeepEqual(s[i], s[0]) {
+			logger.Info("IDP settings mismatch for peer \"%s\": %#v expected same as for peer \"%s\" %#v",
+				peers[i].Name, s[i], peers[0].Name, s[0])
```

Comment:
we should return an error here adding info() in this manner is not useful.

Reply:
Ok done.